app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,18 @@
+
+<h2 class="secondary">Goal-Setting for the Overacheiver Hiding Deep Inside You</h2>
+<ul>
+  <% @tasks.each do |task|%>
+  <li class="task">
+    <%= link_to "#{task.name}", "/tasks/#{task.id}"%> <%if task.completed? %>
+                                                            <%= link_to "Mark Task Incomplete", complete_task_path(task.id), method: 'update', class: 'status' %>
+                                                        <%else%>
+                                                            <%= link_to "Mark Task Complete", complete_task_path(task.id), method: 'update', class: 'status'%>
+                                                        <%end%>
+
+  </li>
+  <%end%>
+</ul>
+
+<h3 class="link">
+    <%= link_to "Add New Task", "/tasks/new"%>
```

Comment:
Best practice would probably make this go to `new_task_path`

Also, watch your indentation so the h3s are aligned